src/services/Content/ContentApi.ts

```diff
@@ -384,7 +384,7 @@ export const getSnapshotList: (
       offset: ((page - 1) * limit).toString(),
       limit: limit?.toString(),
       search,
-      sortBy,
+      sort_by: sortBy,
```

Comment:
Oh is this already in the other PR?

Reply:
oh looks like it is! XD should i close this then and just mark that ticket as done when the other one is merged?